packages/sdk-js/test/features.test.ts

```diff
@@ -484,4 +484,24 @@ describe("features", () => {
     expect(mock.mock.calls.length).toEqual(0);
     window.fetch = f;
   });
+
+  it("considers a feature flag off if coverage is 0", async () => {
+    const growthbook = new GrowthBook({
+      attributes: {
+        foo: "bar",
+      },
+      features: {
+        noCoverage: {
+          rules: [
+            {
+              variations: ["a", "b"],
+              coverage: 0,
+              hashAttribute: "foo",
+            },
+          ],
+        },
+      },
+    });
+    expect(growthbook.isOn("noCoverage")).toEqual(false);
+  });
```

Comment:
We should add this to `cases.json` instead of here.  That way, we can easily share this test case with all of the SDKs.  https://docs.growthbook.io/lib/build-your-own#tests

It makes sense to add this to the `run` tests (starting on line 2103 of cases.json) since it's just about running experiments, not evaluating features.

Reply:
Also, we should make the following updates:

- Bump the latest spec version in `build-your-own.mdx`
- Update the Changelog at the bottom of `build-your-own.mdx`
- Update the spec version at the top of `cases.json`